app/serializers/habitats_serializer.rb

```diff
@@ -0,0 +1,59 @@
+class HabitatsSerializer
+
+  def initialize(habitat, chart_greatest_coverage, chart_protected_areas, global)
+    @habitat = habitat
+    @chart_greatest_coverage = chart_greatest_coverage
+    @chart_protected_areas = chart_protected_areas
+    @global = global
+  end
+
+  def serialize
+    {
+      name: @habitat.name,
+      theme: @habitat.theme,
+      content: content,
+      map: {
+        habitatTitle: @habitat.title,
+        habitatType: @habitat_type,
+        theme: @habitat.theme,
+        tables: [@habitat.poly_table, @habitat.point_table].compact,
+        titleGlobal: @habitat.global_coverage_title(@habitat_type),
+        titleProtected: @habitat.protected_title,
+        percentageGlobal: @habitat.global_coverage,
+        percentageProtected: @habitat.protected_percentage,
+        wmsUrl: @habitat.wms_url
+      },
+      columnChart: @chart_greatest_coverage,
+      rowChart: @chart_protected_areas,
+      disclaimer: @global['disclaimer'],
+      commitments: [
+        aichi_targets, 
+        sdgs,
+        content['other_targets'] 
+      ]
+    }
+  end
+
+  private
+    def content
```

Comment:
Could you please just add an extra line between private and the defitions? Also removing the indentation so it's at the same level as the public methods :)